apidoc/Titanium/Locale/Locale.yml

```diff
@@ -235,3 +251,45 @@ properties:
         sections of wikipedia for reference.
     type: String
     permission: read-only
+
+  - name: systemCountry
```

Comment:
This should be unified via `Ti.Locale.currentCountry`. If something is overridden, it should be reflected in the current country.